mods/taskbar-empty-space-clicks.wh.cpp

```diff
@@ -57,29 +65,46 @@ In case you are using old Windows taskbar on Windows 11 (**ExplorerPatcher** or
   $options:
   - ACTION_NOTHING: Nothing (default)
   - ACTION_SHOW_DESKTOP: Show desktop
-  - ACTION_ALT_TAB: Ctrl+Alt+Tab
+  - ACTION_CTRL_ALT_TAB: Ctrl+Alt+Tab
```

Comment:
That will reset the configuration for uses who chose `ACTION_ALT_TAB`, as now it has a new identifier. Not necessarily too bad, just FYI, up to you whether you prefer a better name or not breaking the settings for users who selected this option.

Reply:
You are right. It is just a minor thing so I partially reverted the change.